### DISCUSSION TITLE
Add a "show_points" option in the ViewerData Class

### DISCUSSION THREAD

**Discussion by xarthurx** [libigl/libigl#1845] [Ideas]
There are `show_lines`, `show_faces` options.

There are `set_edges`, `add_edges` methods.

There are  `set_points`, `add_points` methods.

Since the `edge` series is associated with the `line` option, would it make sense to add an option for `point`, so that I can turn on/off the visualization of the points that are not associated with any mesh?

**Comment by jdumas** [comment]
`show_lines` controls whether or not to show the wireframe of the mesh, not any free overlay line. There is a `show_overlay` parameter that controls whether both overlay lines and points are displayed. I guess we could split both (`show_overlay_lines + show_overlay_points`).

**Reply by xarthurx** [reply]
OK, I've checked the source code of this issue.

the current `show_overlay` option is very integrated with the whole viewer. The tutorials is also using the options a lot...

It might not be a minor change to the code. 
Should we maintain the `show_overlay` option and add the other two? Or other approaches?
